lib/tasks/upgrade.rake

```diff
@@ -283,7 +283,7 @@ namespace :upgrade do
       .order(family_id: :asc, version: :asc, updated_at: :desc)
 
     current_family_id = nil
-    unique_versions = nil
+    unique_versions = Set.new
```

Comment:
The only edge case I can think for this is having templates with family_id set to nil but that should not happen...